skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
-if("${CMAKE_PROJECT_NAME}" STREQUAL "" OR "${CMAKE_PROJECT_NAME}" STREQUAL "platform_se")
-  if(MSVC)
-    set(CMAKE_SYSTEM_VERSION "10.0.17763.0" CACHE STRING INTERNAL FORCE)
-    set(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION "10.0.17763.0" CACHE STRING INTERNAL FORCE)
-  endif()
-  project(platform_se)
-endif()
+project(platform_se	VERSION 2.4.0	LANGUAGES CXX)
```

Comment:
please change the version to 2.3.1
also update this comment https://github.com/skyrim-multiplayer/skymp/blob/main/ci/impl/release.py#L10
and also use spaces in this line, not tabs

Reply:
Haven't found any tabs.
Those empty line changes are trailing spaces removal.